lib/player_choice.rb

```diff
@@ -28,9 +29,9 @@ def is_space_free(board, choice)
 
   def move_confirmation_message(board, choice)
     if !is_user_input_valid(choice)
-      puts "Sorry that is not a valid move. Please select an integer between 1 and 9.\n"
+      @output.sorry_invalid
     elsif !is_space_free(board, choice)
-      puts 'Sorry this space is taken. Please select another.'
+      @output.sorry_taken
```

Comment:
Excellent job moving output-related functionality into its own class and creating well-defined method names!